services/src/main/java/org/exoplatform/news/NewsServiceImpl.java

```diff
@@ -102,6 +105,10 @@
 
   private final static String      PLATFORM_ADMINISTRATORS_GROUP   = "/platform/administrators";
 
+  public static final String       CURRENT_STATE                   = "publication:currentState";
```

Comment:
I think that you can use StageAndVersionPublicationConstant.CURRENT_STATE instead